contrib/vscode/init.sh

```diff
@@ -5,6 +5,15 @@ die () {
 	exit 1
 }
 
+SYSTEM_NAME="$(awk -F= '/^NAME/{print $2}' /etc/os-release)"
+if [ "$SYSTEM_NAME" = "\"Ubuntu\"" ] &&
+   [ ! -e /usr/include/openssl/ssl.h ]; then
```

Comment:
In Git's source code, we prefer `test` over `[`, and `-f` over `-e` for files.

Also, should we not test for more header files than just OpenSSL's? Like, `curl.h`, `expat.h` and `zlib.h`?